app/pages/stacking/delegated-stacking/components/delegated-stacking-info-card.tsx

```diff
@@ -3,19 +3,22 @@ import { BigNumber } from 'bignumber.js';
 
 import { Flex, FlexProps, Text } from '@blockstack/ui';
 
+import { DelegationType } from '@models/index';
 import { Hr } from '@components/hr';
 
 import { toHumanReadableStx } from '@utils/unit-convert';
 import { ExplainerTooltip } from '@components/tooltip';
-import { truncateMiddle } from '../../../../utils/tx-utils';
+import { truncateMiddle } from '@utils/tx-utils';
```

Comment:
`@stacks/ui-utils` has this method

Reply:
![image](https://user-images.githubusercontent.com/1618764/110500183-39363d00-80f9-11eb-8b7b-af0d8d9e3e92.png) 🤔 

package json pointing to non-existent type

Reply:
Sorry try the beta version :Z i need to give it some love